ad7616/system_bd.tcl

```diff
@@ -50,20 +50,18 @@ adi_project_files [list \
 
 source ../../projects/ad7616_sdz/common/ad7616_bd.tcl
 
-if {$SI_OR_PI == 0} {
+if {$SER_PAR_N == 1} {
```

Comment:
Could be simplified to if/else statement. (system_bd/project/tb).

Reply:
Done.